R/zzz.r

```diff
@@ -334,7 +334,8 @@ convert_unsupported_classes <- function(df) {
 }
 
 geojson_rw <- function(input, target = c("char", "list"),
-                       convert_wgs84 = FALSE, crs = NULL, ...){
+                       precision = NULL, convert_wgs84 = FALSE, 
```

Comment:
please move `precision` to the end of the function definition, not a great idea to add new parameters in between old parameters as that might affect other uses of the function

Reply:
thanks for taking the time to take a look at this. I completely agree with adding new parameters to the end of the function call. I made the choice to put it where I did in order to match the syntax of other functions in geojsonio that use the precision argument. However, I agree it is more important to not affect other uses than to keep syntax consistent in all functions. I have committed the change and fixed the test.

Reply:
thanks very much. good reasoning about where you put it in the first place. thanks for changing